### ISSUE TITLE
XXXX (Formal) Projects to Translate button doesn't take you to the formal, but normal page.

### ISSUE THREAD

**Issue by NekoJonez** [Mte90/GlotDict#274]
When I click on "Dutch (Formal) projects to translate" button in plugin/theme translations I expect to come to this page: https://translate.wordpress.org/locale/nl/formal/waiting/
Yet, I go to this page: https://translate.wordpress.org/locale/nl/default/
(Waiting isn't added to the string since we have 0 waiting since recently)
I also tested this with German & German (Formal) and I had the same effect...

**Comment by NekoJonez** [comment]
The issue happens here: 
https://github.com/Mte90/GlotDict/blob/f91d18f97c4ea0b89b82016b85d51003c9548cf9/js/glotdict-functions.js#L102

**Comment by NekoJonez** [comment]
The "/default" should be a variable where it's standard "default" but when it's formal, it should change into formal... 

If only I could code that.

**Comment by Mte90** [comment]
So I see that there is a select menu:
![immagine](https://user-images.githubusercontent.com/403283/123639601-a690e980-d820-11eb-9e81-1612748c914e.png)

we can take the value from there

**Comment by NekoJonez** [comment]
Yeah, since we start from there in most cases right...?

**Comment by Mte90** [comment]
I guess so, I am not localizing anymore and for italian we don't have the formal version so I don't know if there are other difference in the GP UI.

**Comment by NekoJonez** [comment]
The only moment I can see this break (your suggestion) if you don't start from that page with the dropdown but start from a bookmark/link directly from the translation page

**Comment by Mte90** [comment]
If we don't have something where we can get this information from the page itself we cannot do something.
Can I get a link of a page of that case?

**Comment by NekoJonez** [comment]
https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/nl/formal/

If you directly go to that link ...

Now, I noticed something... If you change "projects/wp-plugins/woocommerce/stable" with "locale" you go to the right page... Maybe you can do something with that?

**Comment by Mte90** [comment]
I think that we can extract that info from the URL itself.

**Comment by NekoJonez** [comment]
I don't know php/js at all ... but can't you take the current url and change it...? It's always going to have the locale at the end there... so couldn't you use that as a locale?

Thinking out loud here. (I only know VBA coding so yeah.)

**Comment by Mte90** [comment]
Yes it is my idea to use in that way but I need to study a bit how to reconize the various URLs etc

**Comment by NekoJonez** [comment]
Alright! If I can be of any help researching this... Give me a shout, willing to help :D
